tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36, py37, py38, lint, docker
```

Comment:
@jacobtomlinson We have had to do some hacks here to test the e2ee functionality while waiting for upstream to provide builds. Do you have thoughts on this and the changes to the github actions config file?

Reply:
This feels very matrix specific. Does this mean I will have to run something like `tox -e py38-noe2e` when testing locally?

Reply:
It seems to be a limitation of the gh-actions tox plugin. I have got to what I think is a working compromise, where on the CI we run `noe2e` on windows to override the behaviour of the tox plugin, but locally all `py3x` commands should work just fine.